src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/6.diff.sql

```diff
@@ -0,0 +1,77 @@
+/*************************************************************
+    Search Parameter Status Information
+**************************************************************/
+
+ALTER TABLE dbo.SearchParam
+ADD
+    Status varchar(10) NULL,
+    LastUpdated datetimeoffset(7) NULL,
+    IsPartiallySupported bit NULL
+
+CREATE TYPE dbo.SearchParamTableType_1 AS TABLE
+(
+    Uri varchar(128) COLLATE Latin1_General_100_CS_AS NOT NULL,
+    Status varchar(10) NOT NULL,
+    IsPartiallySupported bit NOT NULL
+)
+
+GO
+
+/*************************************************************
+    Stored procedures for search parameter information
+**************************************************************/
+--
+-- STORED PROCEDURE
+--     GetSearchParamStatuses
+--
+-- DESCRIPTION
+--     Gets all the search parameters and their statuses
```

Comment:
Yes! Added periods in 8e7a49d and 5be0cef.

A bit more info: I was working off of `UpsertResource`, which seems to also be missing a period at the end of the sentence. I am hesitant to change the comment there though, since I think it will break the comment-sensitive schema upgrade tests. Perhaps going back and fixing `UpsertResource` in all schema files can be a change for another PR (..on a slow day haha).